README.md

```diff
@@ -243,11 +243,10 @@ const logger = new ConsoleLogger('SDK', LogLevel.INFO);
 
 // You will need AWS credentials configured before calling AWS or Amazon Chime APIs.
 const chime = new ChimeSDKMessagingClient({ region: 'us-east-1'});
-const endpoint = await chime.send(new GetMessagingSessionEndpointCommand());
 
 const userArn = /* The userArn */;
 const sessionId = /* The sessionId */;
-const configuration = new MessagingSessionConfiguration(userArn, sessionId, endpoint.Endpoint.Url, chime);
+const configuration = new MessagingSessionConfiguration(userArn, sessionId, null, chime);
```

Comment:
`null` -> `undefined` since we are using `undefined` in the method signature.

Reply:
update thx